skore/src/skore/ui/project_routes.py

```diff
@@ -51,6 +56,111 @@ class SerializableProject:
     views: dict[str, Layout]
 
 
+def __cross_validation_item_as_serializable(item: CrossValidationItem) -> dict:
+    # Get tabular results (the cv results in a dataframe-like structure)
+    cv_results = copy.deepcopy(item.cv_results_serialized)
+    cv_results.pop("indices", None)
+
+    tabular_results = {
+        "name": "Cross validation results",
+        "columns": list(cv_results.keys()),
+        "data": list(zip(*cv_results.values())),
+    }
+
+    # Get scalar results (summary statistics of the cv results)
+    def metric_title(metric):
+        m = metric.replace("_", " ")
+        title = f"Mean {m}"
+        if title.endswith(" time"):
+            title = title + " (seconds)"
+        return title
+
+    mean_cv_results = [
+        {
+            "name": metric_title(k),
+            "value": statistics.mean(v),
+            "stddev": statistics.stdev(v),
+        }
+        for k, v in cv_results.items()
+    ]
+
+    scalar_results = mean_cv_results
+
+    # Get estimator details (class name, parameters)
+    _estimator_params = item.estimator_info["params"].items()
+
+    # Figure out the default parameters of the estimator,
+    # so that we can highlight the non-default ones in the UI
+
+    # This is done by instantiating the class with no arguments and
+    # computing the diff between the default and ours
+    try:
+        estimator_module = importlib.import_module(item.estimator_info["module"])
+        EstimatorClass = getattr(estimator_module, item.estimator_info["name"])
+        default_estimator_params = {
+            k: repr(v) for k, v in EstimatorClass().get_params().items()
+        }
+    except Exception:
+        default_estimator_params = {}
```

Comment:
Note with hindsight: to avoid environment dependency, we should computes these fields in the item. We can't assure that the module is still available during export backend to frontend.

Reply:
I trust `get_params()` more than the signature, since it is a public method.

About computing the fields in the item: yes, that sounds much better, we should do it!

Reply:
I don't understand your point on `get_params`. 
The signature of a function is fixed. If you want to check for non-default parameters, the signature is sufficient.
`get_params` can have side-effect on parameters, and you risk highlighting parameters that are not passed by user, but introduced by `get_params`. User can be confused.

I would prefer to store the real passed parameters in `estimator_info["params"]` and retrieve here the default parameters using signature in `default_estimator_params`.

Reply:
And moreover,  `get_params` is only available for `scikit-learn`, but below (lines 119-127), you assume that you can received a model from a non `scikit-learn` module.

Reply:
However, we do assume the estimator is from sklearn [in CrossValidationItem's factory](https://github.com/probabl-ai/skore/blob/a59f0378f4d3b741db1c3b39d8c695528d64ca8e/skore/src/skore/item/cross_validation_item.py#L142) which is wrong!

Reply:
On second thought, since this factory is only used to serialize information around a wrapper to sklearn's `cross_validate`, I think that we can reasonably restrict this to sklearn-compatible estimators. Still, this is a potential error that should be dealt with.

Reply:
> If you want to check for non-default parameters, the signature is sufficient

We need `get_params` in general to get the parameters (e.g. to deal with sklearn `Pipeline`s, which don't have a fixed signature), so I don't think experimenting with the signature is worth it at the moment.

Reply:
See #970